charts/cluster-proxy/values.yaml

```diff
@@ -12,8 +12,8 @@ replicas: 1
 
 spokeAddonNamespace: "open-cluster-management-cluster-proxy"
 
-proxyServerImage: cluster-proxy
-proxyAgentImage: cluster-proxy
+proxyServerImage: quay.io/open-cluster-management/cluster-proxy:v0.1.4
```

Comment:
how about we merge the image values into one?

Reply:
Sure we can do it. Do we need to update the API structure as well?

Reply:
Old:
```
proxyServerImage: quay.io/open-cluster-management/cluster-proxy:v0.1.4
proxyAgentImage: quay.io/open-cluster-management/cluster-proxy:v0.1.4
```

New:
```
ANPImage: quay.io/open-cluster-management/cluster-proxy:v0.1.4
```
When we say "merge into one", Do we mean modification as the above?

Reply:
oh the point of this pull is to separate the addon image and ANP's image, i was talking about converging all the images into one above. nvmd, lets get this in.